src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+
+  const { slug = '' } = useParams();
+
+  const isPeople = !isLoading && !hasError && people.length > 0;
+  const isEmptyTable = !isLoading && !hasError && people.length === 0;
```

Comment:
these two variables means absolutely the same, you just leave one of them, and use `!` where needed